frontend/src/interfaces/uiConfig.ts

```diff
@@ -53,6 +53,7 @@ export interface IFlags {
     customRootRoles?: boolean;
     strategyVariant?: boolean;
     newProjectLayout?: boolean;
+    configurableFeatureTypeLifetimes?: boolean;
```

Comment:
We might want to unify this in the future to have a single source of for flags available in frontend

Reply:
You mean as in unifying this file and the `experimental.ts` file we have in the server source code?

Reply:
yes, or automating it in some capacity